src/lib.rs

```diff
@@ -87,7 +87,7 @@ fn should_use_ctru_debuginfo(cargo_cmd: &Command, verbose: bool) -> bool {
             let Some(unit) = unit_graph
                 .units
                 .iter()
-                .find(|unit| unit.target.name == "ctru-sys")
+                .find(|unit| unit.target.name == "ctru_sys")
```

Comment:
oops, guess this actually just ... never worked properly?

Reply:
That… may have been the case (it’s also my fault for not catching this).